src/components/link/ActionLink.js

```diff
@@ -13,7 +13,7 @@ const propTypes = {
   /**
    * Title attribute.
    */
-  title: PropTypes.string.isRequired,
+  title: PropTypes.string,
```

Comment:
I think I put it 'required' because, for the  "barrierefreiheit", we HAVWE TO put a title  and ariaLabel everywhere, that avoid the devloper to forget about this.
I think we should discuss this kind of things on monday.